web/src/components/utils/axios_errors_handlers/AxiosErrorHander.ts

```diff
@@ -0,0 +1,9 @@
+import Axios from "axios";
+
+const axiosErrorExceptionHandler = (err: Error) => {
+    if (!Axios.isCancel(err)) {
+        console.log(err);
+      }
+}
+
+export default axiosErrorExceptionHandler;
```

Comment:
Usually, ending files without a new line is bad style.

Can you check prettifier configuration because it should be fixed on the fly and also checked by CI (https://github.com/ryukzak/nitta/runs/3680547729?check_suite_focus=true#step:2:11)?

Reply:
Looks like path to files, which need to be checked by prettier, should be changed to `yarn exec -s prettier -- --check src/**/**/**/*.{ts,tsx}` instead of `yarn exec -s prettier -- --check src/**/*.{ts,tsx}`. Better to use more general solution but I don't have any idea. Should I commit this change?

Reply:
Or `yarn exec -s prettier -- --check src/**/**/*.{ts,tsx}`, if I move AxiosErrorHadnlers.ts to util (as suggested below). But then inner files of gen folder will also be checked, so I need to manually list necessary dirs. It's not simple question how to do it better

Reply:
You can ignore 'gen' by the bang. 

prettier-eslint --write "{,!(node_modules)/**/}*.{js,jsx}"

If I don't mistake, ** means all folders and subfolders.

Reply:
Ah, current command work almost properly (except for 'gen') if you put double quotes around path, like `yarn exec -s prettier -- --check "src/**/*".{ts,tsx}` 

Thank you for source, I found a way to ignore 'gen' directory `yarn exec -s prettier -- --check "src/**/{,!(gen)}/*.{ts,tsx}"`